circup/__init__.py

```diff
@@ -636,20 +1248,20 @@ def get_volume_name(disk_name):
     return device_dir
 
 
-def find_modules(device_path, bundles_list):
+def find_modules(backend, device_url, bundles_list):
```

Comment:
I'd expect this to only take the backend in. It shouldn't need to know a separate path.